premake

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+# 
+# This script allows the sketcher repo to be built using buildinger, the
+# Schrodinger internal build system
+#
+
+. $SCHRODINGER_SRC/mmshare/build_env
+set -euo pipefail
```

Comment:
@d-b-w  -- is there anything in this that we wouldn't want checked into a public repository? I assume it's fine, but it does feel weird to bleed our interns out into the world